source/_integrations/alarm_control_panel.mqtt.markdown

```diff
@@ -56,6 +56,11 @@ availability:
       description: An MQTT topic subscribed to receive availability (online/offline) updates.
       required: true
       type: string
+availability_mode:
+  description: When `availability` is configured, this controls the conditions needed to set the entity to `available`. Valid entries are `all`, `any`, and `latest`. If set to `all` a valid `payload_available` must be received on all configured availability topics before the entity is marked as online. If set to `any` a valid `payload_available` must be received on at least one configured availability topics before the entity is marked as online. If set to `latest`, the last valid `payload_available` or `payload_not_available` received on any configured availability topic controls the availability.
+  required: false
+  type: string
+  default: latest
```

Comment:
In my opinion this describes the logic well.

Some textual suggestions:
- If set to `all` a valid `payload_available`        -> If set to `all` a `payload_available`
- If set to `any` a valid `payload_available` -> If set to `any` a `payload_available`
- one configured availability topics -> one configured availability topic      (without the trailing s)
- the last valid payload_available -> the last valid payload_available
- the last valid payload_available -> the last payload_available

Reply:
Thanks! Updated.